website/docs/04_comparisons/01_helm.md

```diff
@@ -0,0 +1,27 @@
+---
+slug: helm
+sidebar_label: With Helm
+title: Glasskube vs Helm Comparison
+description: Discover Glasskube, a fresh Kubernetes package manager challenging Helm's shortcomings. Explore automatic updates, dependency management and much more.
```

Comment:
The description and the ordered list (second comment) is new the other paragraphs did not change (much), but due to the new file name GitHub doesn't show a meaningful side-by-side view.